plugins/kubernetes/app/models/watchers/cluster_pod_error_watcher.rb

```diff
@@ -2,34 +2,38 @@ module Watchers
   class ClusterPodErrorWatcher < BaseClusterWatcher
     include Celluloid::Notifications
 
```

Comment:
Missing `include Watchers::TopicSubscription`

This watcher will crash on each attempt to publish to a topic. I had my reservations towards that mixin and my hunch was confirmed when I tried running some ad-hoc tests. Maybe it would be safer not to include that module but call the method directly?